test/extensions/filters/network/common/redis/BUILD

```diff
@@ -0,0 +1,36 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_mock",
+    "envoy_package",
+)
+load(
+    "//test/extensions:extensions_build_system.bzl",
+    "envoy_cc_test",
+    "envoy_extension_cc_test_binary",
```

Comment:
This is not used, I would remove this entire load() and just import the normal envoy_cc_test from the normal build size bzl file.

Reply:
done